src/components/Exchange/constants.ts

```diff
@@ -0,0 +1,11 @@
+export enum ErrorCodes {
+  poolExceeded = "POOL_EXCEEDED",
+  buffer = "BUFFER",
+  maxUSDG = "MAX_USDG",
+  invalidLiqPrice = "INVALID_LIQ_PRICE",
+}
+
+export enum ErrorTypes {
```

Comment:
For me personally ErrorType sounds like the type of the error and not how we show it

I would call it ErrorDisplayType - but that's as you like it)

Reply:
Updated